content/en/flux/cheatsheets/oci-artifacts.md

```diff
@@ -337,6 +337,93 @@ For more details on how to setup contextual authorization for Azure, AWS and Goo
 - [OCIRepository documentation](/flux/components/source/ocirepositories/#provider)
 - [HelmRepository documentation](/flux/components/source/helmrepositories/#provider)
 
+## Signing and verification
+
+Starting with v0.35, Flux comes with support for verifying OCI artifacts
+signed with [Sigstore Cosign](https://github.com/sigstore/cosign).
+
+To secure your delivery pipeline, you can sign the artifacts and configure Flux
+to verify the artifacts' signatures before they are downloaded and reconciled in production.
+
+### Workflow example
+
+Generate a Cosign key-pair and create a Kubernetes secret with the public key:
+
+```shell
+cosign generate-key-pair
+
+kubectl -n flux-system create secret generic cosign-pub \
+  --from-file=cosign.pub=cosign.pub
+```
+
+Push and sign the artifact using the Cosign private key:
+
+```shell
+flux push artifact oci://ghcr.io/stefanprodan/manifests/podinfo:$(git tag --points-at HEAD) \
```

Comment:
Maybe replace `ghcr.io/...` here with `<image reference>`? I see it is also used in the examples above, but would think it is easier for people to follow when they have guidance on what needs to be replaced. Instead of accidentally copying the command in full, then discovering they can not push to this image repository, to then figure out what needs replacing.